### PR TITLE
set `@wrappers` and `@env_shebang` to true

### PR DESCRIPTION
**Pull request by tenderlove**
these conditionals would always evaluate to `true`, so just directly set
them to true without checking the option value

### CODE REVIEW

File: lib/bundler/source/path/installer.rb [hunk 1]
```diff
@@ -11,8 +11,8 @@ def initialize(spec, options = {})
           @gem_bin_dir       = "#{Bundler.rubygems.gem_dir}/bin"
           @bin_dir           = Bundler.requires_sudo? ? @tmp_bin_dir : @gem_bin_dir
           @gem_dir           = Bundler.rubygems.path(spec.full_gem_path)
-          @wrappers          = options[:wrappers] || true
-          @env_shebang       = options[:env_shebang] || true
+          @wrappers          = true
```
Comment (segiddins) [inline lib/bundler/source/path/installer.rb]: maybe this should be `options.fetch(:wrappers, true)`?

Comment (indirect) [inline lib/bundler/source/path/installer.rb]: I'd rather not start allowing it to be set if we haven't allowed it this entire time so far....

Comment (segiddins) [inline lib/bundler/source/path/installer.rb]: It seems we were trying to set it to `false` in `path.rb`

Comment (indirect) [inline lib/bundler/source/path/installer.rb]: Given that it changed 5 years ago in 93033523 and no one noticed, I think we should probably keep it the way it is now.

**Comment by segiddins** [comment]
The single failure looks like a Travis issue, so this should be good

**Pull request merged**